app/src/main/java/com/samco/trackandgraph/antlr/evaluation/Evaluation.kt

```diff
@@ -0,0 +1,99 @@
+package com.samco.trackandgraph.antlr.evaluation
+
+
+import com.samco.trackandgraph.antlr.ast.*
+import com.samco.trackandgraph.antlr.parsing.DatatransformationFunctionParserFacade
+import com.samco.trackandgraph.database.entity.DataPoint
+
+fun inferDatatype(data: List<DataPoint>) : DataType {
+    return DataType.NUMERICAL
+}
+
+class EvaluationModel {
+
```

Comment:
The main function to be called. Checks for parse errors first then then evaluates the code. 
Each line consists of a Statement and so we evaluate them one after the other